.github/workflows/turbopack-nextjs-build-integration-tests.yml

```diff
@@ -1,212 +1,24 @@
-      matrix:
-        group: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
-
-    steps:
-      - name: Setup Node.js
-        uses: actions/setup-node@v4
-        with:
-          node-version: 18.18.2
-          check-latest: true
-      - uses: actions/cache/restore@v3
-        id: restore-build
-        with:
-          path: ./*
-          key: ${{ inputs.version || 'canary' }}-${{ github.sha }}
-          fail-on-cache-miss: true
-
-      - name: Enable corepack and install yarn
-        run: |
-          corepack enable
-          corepack prepare --activate yarn@1.22.19
-
-      - name: Setup playwright
-        run: |
-          pnpm playwright install
-
-      - name: Run test/integration
-        run: |
-          TURBOPACK=1 TURBOPACK_BUILD=1 NEXT_E2E_TEST_TIMEOUT=240000 node run-tests.js -g ${{ matrix.group }}/12 -c ${TEST_CONCURRENCY} --type integration
-        continue-on-error: true
-
-      - name: Upload test report artifacts
-        uses: actions/upload-artifact@v4
-        with:
-          name: test-reports-build-integration-${{ matrix.group }}
-          if-no-files-found: 'error'
-          path: |
-            test/turbopack-test-junit-report
-
-  # Collect integration test results from execute_tests,
-  # Store it as github artifact for next step to consume.
-  collect_nextjs_production_integration_stat:
-    needs: [test-production, test-integration-production]
-    name: Next.js integration test production status report
-    runs-on:
-      - 'self-hosted'
-      - 'linux'
-      - 'x64'
-      - 'metal'
-
-    if: always()
-    permissions:
-      pull-requests: write
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-
-      - name: Collect integration test stat
-        uses: ./.github/actions/next-integration-stat
-        with:
-          diff_base: ${{ inputs.diff_base || 'none' }}
-
-      - name: Store artifacts
-        uses: actions/upload-artifact@v4
-        with:
-          name: test-results-turbopack-production
-          path: |
-            nextjs-test-results.json
-            failed-test-path-list.json
-            passed-test-path-list.json
-
-  upload_test_report:
-    needs: [test-production, test-integration-production]
-    name: Upload test report to datadog
-    runs-on:
-      - 'self-hosted'
-      - 'linux'
-      - 'x64'
-      - 'metal'
-
-    if: ${{ !cancelled() }}
-    steps:
-      - name: Download test report artifacts
-        id: download-test-reports
-        uses: actions/download-artifact@v4
-        with:
-          pattern: test-reports-*
-          path: test/reports
-          merge-multiple: true
-
-      - name: Upload to datadog
-        env:
-          DATADOG_API_KEY: ${{ secrets.DATA_DOG_API_KEY }}
-          DD_ENV: 'ci'
-        run: |
-          # We'll tag this to the "Turbopack" datadog service, not "nextjs"
-          npx @datadog/datadog-ci@2.23.1 junit upload --tags test.type:turbopack-build.daily --service Turbopack-build ./test/reports
```

Comment:
https://vercel.slack.com/archives/C03EWR7LGEN/p1740178181767749

This is now redundant with the `build_reusable` workflow's datadog step: https://github.com/vercel/next.js/blob/canary/.github/workflows/build_reusable.yml#L246-L264